### PR TITLE
increased min_track_length from 2 to 3 in config files

### PR DESCRIPTION
**Pull request by travisdriver**
As per last weeks meeting

### CODE REVIEW

**Review by ayushbaid — APPROVED** [review]

**Comment by ayushbaid** [comment]
Please document this change with metrics. Or you can wait until I create notebooks to run it with all 4 datasets.

**Comment by ayushbaid** [comment]
Metrics for `DeepFrontEnd` on the door dataset:

| Metric | Baseline | 2-view BA |
| ------ | --------- | --------- |
| Input number of tracks | 4904 | 3978 |
| Output number of tracks | 4579 | 3651 | 
| Mean track length | 7.016 | 8.337 | 
| Mean reproj error | 0.592 | 0.604 |

**Comment by ayushbaid** [comment]
@travisdriver if you have the metrics for this, lets discuss this tomorrow in the meeting?

**Comment by travisdriver** [comment]
You posted the metrics for the crane mast dataset. Did you want metrics from the asteroid datasets?

**Comment by johnwlambert** [comment]
I think we need to pull in master (or just create a fresh PR since the conflicts are so large), and then Github Actions will run and upload the HTML reports for the 6 dataset/config pairs. Then we should analyze each one w.r.t. pose errors, etc

**Comment by travisdriver** [comment]
It should be as easy as pulling from master, as I only changed the config files. I'll do that now.

**Comment by travisdriver** [comment]
Oh nevermind, I guess you guys changed other files.

**Comment by johnwlambert** [comment]
Hi @travisdriver, can you pull master into one more time? more bug fixes yesterday were landed that affect this (like the histogram plot in the HTML report)

File: environment_mac.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: gtsfm-v1
 channels:
-  # The GTSFM mac environment closely follows the linux conda environment file, 
+  # The GTSFM mac environment closely follows the linux conda environment file,
```
[the remark this answers is not in the document]
Comment (travisdriver) [reply environment_mac.yml:3]: It causes a mypy error, but I can change it back if you want. Let me know.

[the remark this answers is not in the document]
Comment (johnwlambert) [reply environment_mac.yml:3]: Ok, let's keep then. Thanks

**Comment by johnwlambert** [comment]
i tried to pull master into this branch, but got like 8 merge conflicts. I think it might be easier and cleaner for the git history just to start with a clean branch off master and make the few changes

**Comment by travisdriver** [comment]
Alright, I'll do this soon and close this branch once I do.

**Comment by ayushbaid** [comment]
Closing in favor of #260

**Pull request closed without merging**